recipes/perl/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import chdir, copy, get, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.54.0"
+
+class PerlConan(ConanFile):
+    name = "perl"
+    description = "A high-level, general-purpose, interpreted, dynamic programming language"
+    license = "GPL-1.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.perl.org/"
+    topics = ("scripting", "programming", "language")
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def configure(self):
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def validate(self):
+        if self.settings.os == "Windows":
```

Comment:
The recipe might work with MinGW. Maybe restrict it for MSVC only.

Reply:
I sort of just assumed it didn't work, but I didn't actually try. I'll give it a shot and see how it goes.

Reply:
Still working on this on the side - Seems they at least officially support MinGW which is neat. No surprise though that the Windows build of a Makefile-based project is difficult to work with.